app/src/main/java/com/claraberriel/mvpparking/fragments/ReleaseFragment.java

```diff
@@ -1,23 +1,58 @@
 package com.claraberriel.mvpparking.fragments;
 
+import android.content.Context;
 import android.os.Bundle;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 
+import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
 
-import com.claraberriel.mvpparking.R;
+import com.claraberriel.mvpparking.databinding.FragmentReleaseBinding;
+import com.claraberriel.mvpparking.entities.Parking;
+import com.claraberriel.mvpparking.mvp.model.ReleaseModel;
+import com.claraberriel.mvpparking.mvp.presenter.ReleasePresenter;
+import com.claraberriel.mvpparking.mvp.view.ReleaseView;
 
 public class ReleaseFragment extends Fragment {
+    public static final String PARKING_ARGUMENT = "PARKING";
+    private ReleaseFragmentDelegate delegate;
+    private ReleasePresenter releasePresenter;
+    private FragmentReleaseBinding fragmentReleaseBinding;
 
-    public ReleaseFragment() {
+
+    public interface ReleaseFragmentDelegate {
+        void onReleaseFragmentButtonClicked(Parking parking);
+    }
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+        if (context instanceof ReleaseFragmentDelegate) {
+            delegate = (ReleaseFragmentDelegate) context;
+        }
     }
 
     @Nullable
     @Override
-    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_release, container, false);
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        fragmentReleaseBinding = FragmentReleaseBinding.inflate(inflater, container, false);
+
+        if(getArguments() != null) {
+            Parking parking = getArguments().getParcelable(PARKING_ARGUMENT);
+            releasePresenter = new ReleasePresenter(new ReleaseModel(parking), new ReleaseView(this, fragmentReleaseBinding));
+        }
+        setListeners();
```

Comment:
This could cause a crash because of NPE when arguments are null, move inside IF.